src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,75 @@
+.movie-card {
+  height: 90%;
+  width: auto;
+  padding: 0;
+  margin: 20px;
+  border-radius: 5px;
+  position: relative;
+  padding-left: 10px;
+  align-content: center;
+  font-size: 0.8em;
+  font-family: sans-serif;
+  color: whitesmoke;
+  font-kerning: normal;
+  text-shadow: 0ch;
+  outline-color: black;
```

Comment:
```suggestion
 
```

Reply:
I didn't see that this was affecting any outlines, again maybe I was missing it.